test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -60,6 +65,22 @@ if (OSSFUZZ)
     )
     set_target_properties(yul_proto_diff_ossfuzz PROPERTIES LINK_FLAGS ${LIB_FUZZING_ENGINE})
 
+    add_executable(yul_proto_diff_custom_mutate_ossfuzz
+            yulProto_diff_ossfuzz.cpp
+            yulFuzzerCommon.cpp
+            protoToYul.cpp
+            yulProto.pb.cc
+            protomutators/YulProtoMutator.cpp
+)
+    target_include_directories(yul_proto_diff_custom_mutate_ossfuzz PRIVATE /usr/include/libprotobuf-mutator)
```

Comment:
Not sure if this was discussed already, but isn't there a better way than to hardcode `/usr/include/libprotobuf-mutator`? It's also used above. Maybe unrelated to this PR.

Reply:
True. I think this can easily be done once I can pinpoint where in the docker image LPM is installed. In fact, may be the `include` directive may no longer be required as well if it is in a standard system wide location.

I will create an issue for this.